### ISSUE TITLE
keyword-only arguments not allowed after varargs when using ParamSpec

### ISSUE THREAD

**Issue by zhykzhykzhyk** [microsoft/pyright#4974]
**Describe the bug**
keyword-only arguments not allowed after varargs when using ParamSpec.

**To Reproduce**
Run pyright with following program, got errors
```
c.py:18:21 - error: No parameter named "default" (reportGeneralTypeIssues)
c.py:19:21 - error: No parameter named "default" (reportGeneralTypeIssues)
```

**Expected behavior**
No errors.

**Screenshots or Code**
If applicable, add screenshots or the text of the code (surrounded by triple back ticks) to help explain your problem.
```python
# pyright: strict

from typing import Callable, TypeVar, ParamSpec

_T = TypeVar("_T")
_P = ParamSpec("_P")


def f(*args: int, default: int = 1) -> int:
    return default


def apply(fn: Callable[_P, _T], /, *args: _P.args, **kwargs: _P.kwargs) -> _T:
    return fn(*args, **kwargs)


def g(*args: int):
    apply(f, *args, default=5)  # expect no error, got 'No parameter named "default"'
    apply(f, *[1, 2, 3], default=5)  # expect no error, got 'No parameter named "default"'

    f(*args, default=5)  # this is ok
    apply(f, 1, 2, 3, default=5)  # this is also ok
```

**VS Code extension or command-line**
pyright 1.1.303

**Additional context**
None

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.305, which I just published. It will also be included in a future release of pylance.
